sqlglot/dialects/hive.py

```diff
@@ -485,7 +486,7 @@ class Generator(generator.Generator):
                 [
                     transforms.eliminate_qualify,
                     transforms.eliminate_distinct_on,
-                    transforms.unnest_to_explode,
+                    partial(transforms.unnest_to_explode, unnest_using_arrays_zip=False),
```

Comment:
Did you confirm that HiveQL does not have `ARRAYS_ZIP`? Usually, if something works for Spark 2 (which has both `INLINE()` and `ARRAYS_ZIP()`) we consider it a part of the Hive dialect too.

Reply:
I am in hive3.1, there is no ARRAYS_ZIP